include/COLA.hh

```diff
@@ -185,7 +193,8 @@ namespace cola {
         ~MetaProcessor() = default;
 
         void reg(std::shared_ptr<VFactory> factory, const std::string& name, const std::string& type);
-        FilterAnsamble parse(const MetaData& data);
+        FilterAnsamble parse(const SimpleMetaData& data);
```

Comment:
I don't see any usecase for SimpleMetaData struct. Maybe remove it entirely?

Reply:
I will remove it just before the approval